app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/guidance.scala.html

```diff
@@ -70,7 +83,7 @@
           s"""
            | <p>${messages("multiple-disposals.guidance.details.p1")}</p>
            | <p>${messages("multiple-disposals.guidance.details.p2")}</p>
-           | <p>${messages(s"multiple-disposals.guidance.details$userKey.link", appConfig.tranferringOwnershipHelp)}</p>
+           | <p>${messages(s"multiple-disposals.guidance.details.link", appConfig.tranferringOwnershipHelp)}</p>
```

Comment:
Im seeing different keys to agents, trusts etc for "link" text.

Reply:
Is it expected behaviour?

Reply:
yes texts are like "Check the person's residence status..." and "Check your client's residence status..."

Reply:
ah hold on that's a different link - let me check

Reply:
the content is identical for all users so I removed userKey - previously the wrong link text was used - now all users get same link `Read about ‘exchange’ and ‘completion’ (opens in a new window)` which has no user customisation

Reply:
I should point out I removed the link versions for agent and trust from message file (they were wrong text anyway for this link)